### PR TITLE
fix jit on mac

### LINKED ISSUE #14271: jit在mac上编译问题
image

### PR DESCRIPTION
**Pull request by tensor-tang**
fix #14271 

test=develop

### CODE REVIEW

File: paddle/fluid/operators/math/jit_kernel_blas.cc [hunk 1]
```diff
@@ -14,10 +14,13 @@ limitations under the License. */
 
 #include "paddle/fluid/operators/math/jit_kernel.h"
 #include <string>
-#include "paddle/fluid/operators/math/jit_code.h"
 #include "paddle/fluid/operators/math/jit_kernel_macro.h"
 #include "paddle/fluid/platform/enforce.h"
 
+#ifdef PADDLE_WITH_XBYAK
```
Comment (gongweibao) [inline paddle/fluid/operators/math/jit_kernel_blas.cc:20]: Who will define it?

Comment (tensor-tang) [reply paddle/fluid/operators/math/jit_kernel_blas.cc:20]: https://github.com/PaddlePaddle/Paddle/blob/8fc05e0373bb481e36b53c650f2dc00acf1b32a5/cmake/external/xbyak.cmake#L15-L19 

Let's why failed on Mac

Comment (gongweibao) [reply paddle/fluid/operators/math/jit_kernel_blas.cc:20]: OK

Comment (gongweibao) [reply paddle/fluid/operators/math/jit_kernel_blas.cc]: OK

**Comment by heavengate** [comment]
Build success in my Mac after pick this commit

**Comment by tensor-tang** [comment]
Thanks @heavengate

**Review by gongweibao — APPROVED** [review]
LGTM

**Pull request merged**